actors/builtin/power/power_test.go

```diff
@@ -329,29 +326,29 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 		actor.createMinerBasic(rt, owner, owner, miner4)
 		actor.createMinerBasic(rt, owner, owner, miner5)
 
-		actor.updateClaimedPower(rt, miner1, div(smallPowerUnit, 2), smallPowerUnit)
-		actor.updateClaimedPower(rt, miner2, div(smallPowerUnit, 2), smallPowerUnit)
+		actor.updateClaimedPower(rt, miner1, smallPowerUnit, smallPowerUnit)
```

Comment:
Perhaps multiply the QA power by 10 so it's more clear that it's not being used? Ensure the QA power does exceed the threshold, but is ignored.